### PR TITLE
Create 2021-75.md

### PR DESCRIPTION
**Pull request by billfox1**
Release Notes for 7.14

### CODE REVIEW

File: release-notes/2021-75.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Integration Updates
+
+### Artifactory
+- Allow user to configure optional ingestion of pipeline resources
+
+### AWS
+- Fix AWS API throttling in IAM processing step
+- Reduce number of AWS API calls made during IAM policy processing
+- Reduce number of AWS API calls made during EC2, Inspector, and ECR processing
+- Improve processing of security groups and build relationships to protected resources (`aws_security_group - PROTECTS -> resource`)
```
Comment (ninetreats) [inline release-notes/2021-75.md:36]: should it be `aww_security_group -> PROTECTS -> resource` ?

**Review by ninetreats — DISMISSED** [review]
added one question on line 36.

File: release-notes/2021-75.md (hunk 1 above)
Comment (billfox1) [reply release-notes/2021-75.md:36]: It renders like that via Preview, not in source

Comment (ninetreats) [reply release-notes/2021-75.md:36]: it is missing a >

Comment (ninetreats) [reply release-notes/2021-75.md:36]: or actually looking at previous release notes, it should be `aws_security_group` `PROTECTS` `resouces` to be consistent.

Comment (billfox1) [reply release-notes/2021-75.md]: OK, made the changes you've suggested above

File: release-notes/2021-75.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+2021-07-14
+
+## New Features and Improvements
+
+- Added support for a markdown widget to the Insights app
+
+- Updated the Owners & Maintainers tab in the Vulnerabilities view to show the query that returned no results if nothing was found
+
+- Updated the look & feel of the Policies app
```
Comment (erkangz) [inline release-notes/2021-75.md:11]:
```suggestion
- Improved UI/UX of the **Policies** app with a more intuitive layout of the 
  domains/policies and the controls/procedures for each domain.
```

File: release-notes/2021-75.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# JupiterOne 2021.75 Release
+
+2021-07-14
+
+## New Features and Improvements
+
+- Added support for a markdown widget to the Insights app
```
Comment (erkangz) [inline release-notes/2021-75.md:7]:
```suggestion
- Added a **Markdown widget** to the **Insights** app that allows static HTML content 
  to be added to any Insights board.
```

File: release-notes/2021-75.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+- Updated the Owners & Maintainers tab in the Vulnerabilities view to show the query that returned no results if nothing was found
+
+- Updated the look & feel of the Policies app
+
+- Customers who have purchased the "Powerup Pack" or are at the Enterprise tier can now upload a custom logo for their J1 instance
```
Comment (erkangz) [inline release-notes/2021-75.md:13]:
```suggestion
- Customers who have purchased the "Powerup Pack" or are at the Enterprise tier can 
  now upload a **custom logo** for their J1 instance. The custom logo will be
  displayed in the main app UI and used in the policies PDF export.
```

File: release-notes/2021-75.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## New Integrations
+
+### Kubernetes Beta
+We have beeen working on an [integration for Kubernetes](https://github.com/JupiterOne/graph-kubernetes/blob/master/docs/jupiterone.md) clusters. [Please reach out](https://forms.gle/HoPsNdtMPwdzVA367) if you're interested in using the integration and help us make it valuable for your operations!
```
Comment (erkangz) [inline release-notes/2021-75.md:19]: insert blank line after heading. same for the rest of the doc.

File: release-notes/2021-75.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+# JupiterOne 2021.75 Release
+
+2021-07-14
+
+## New Features and Improvements
+
+- Added support for a markdown widget to the Insights app
+
+- Updated the Owners & Maintainers tab in the Vulnerabilities view to show the query that returned no results if nothing was found
```
Comment (erkangz) [inline release-notes/2021-75.md:9]:
```suggestion
- Updated the **Owners & Maintainers** tab in the **Vulnerabilities** view to show the 
  query if no results were returned.
```

**Review by erkangz — CHANGES_REQUESTED** [review]

**Review by AnneNielsenJ1 — DISMISSED** [review]
no further changes from me. Thanks!

**Review by ninetreats — APPROVED** [review]

**Review by erkangz — APPROVED** [review]

**Pull request merged**